helpers/virtual/virtual.go

```diff
@@ -38,23 +38,28 @@ func UpgradeVirtualGuest(
 	when ...time.Time,
 ) (datatypes.Container_Product_Order_Receipt, error) {
 
-	if guest.PrivateNetworkOnlyFlag == nil || guest.DedicatedAccountHostOnlyFlag == nil {
-		service := services.GetVirtualGuestService(sess)
-		guestForFlag, err := service.Id(*guest.Id).Mask("privateNetworkOnlyFlag,dedicatedAccountHostOnlyFlag").GetObject()
-		if err != nil {
-			return datatypes.Container_Product_Order_Receipt{}, err
-		}
-
-		guest.PrivateNetworkOnlyFlag = guestForFlag.PrivateNetworkOnlyFlag
-		guest.DedicatedAccountHostOnlyFlag = guestForFlag.DedicatedAccountHostOnlyFlag
-	}
+	// Get the packageID of the guest and use same during upgrade of guest.
 
-	pkg, err := product.GetPackageByType(sess, "VIRTUAL_SERVER_INSTANCE")
+	service := services.GetVirtualGuestService(sess)
+	guestPackage, err := service.Id(*guest.Id).Mask("privateNetworkOnlyFlag,dedicatedAccountHostOnlyFlag,billingItem[package[id]]").GetObject()
 	if err != nil {
 		return datatypes.Container_Product_Order_Receipt{}, err
 	}
 
-	productItems, err := product.GetPackageProducts(sess, *pkg.Id)
+	if guest.PrivateNetworkOnlyFlag == nil || guest.DedicatedAccountHostOnlyFlag == nil {
+
+		guest.PrivateNetworkOnlyFlag = guestPackage.PrivateNetworkOnlyFlag
+		guest.DedicatedAccountHostOnlyFlag = guestPackage.DedicatedAccountHostOnlyFlag
+	}
+
+	var packageID int
+
+	if guestPackage.BillingItem != nil {
+
```

Comment:
why the blank lines? has this been run through gofmt?

Reply:
After running with gofmt -w helpers/virtual/virtual.go I don't see any changes but anyways I removed the blank lines..Can you review?